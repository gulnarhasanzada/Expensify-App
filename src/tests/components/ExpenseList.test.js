import React from "react";
import { ExpenseList } from "../../components/ExpenseList";
import expenses from "../fixtures/expenses";
import { shallow } from "enzyme";

test('Should render Expense List with expenses',()=>{
    const wrapper = shallow(<ExpenseList expenses = {expenses}/>);
    expect(wrapper).toMatchSnapshot();
});

test('Should render Expense List with empty array', () => {
    const wrapper = shallow(<ExpenseList expenses={[]} />);
    expect(wrapper).toMatchSnapshot();
});