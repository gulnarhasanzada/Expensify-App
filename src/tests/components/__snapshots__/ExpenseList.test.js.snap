// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Expense List with empty array 1`] = `
<div>
  <h1>
    Expense List
  </h1>
  <p>
    No expenses
  </p>
</div>
`;

exports[`Should render Expense List with expenses 1`] = `
<div>
  <h1>
    Expense List
  </h1>
  <ExpenseListItem
    amount={197}
    createdAt={-345600000}
    description="Rent"
    id="1"
    key="1"
    note="Rent for august"
  />
  <ExpenseListItem
    amount={3.5}
    createdAt={345600000}
    description="Coffee"
    id="2"
    key="2"
    note="Starbucks"
  />
  <ExpenseListItem
    amount={10}
    createdAt={0}
    description="Book"
    id="3"
    key="3"
    note="Book buy"
  />
</div>
`;
